.travis_target.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -ev
```

Comment:
I stripped down this file compared to Facter and Leatherman. I'm not sure if I removed anything important (libSegFault?, c++filt?).